src/app/header/header.component.ts

```diff
@@ -18,21 +18,21 @@ export class HeaderComponent implements OnInit {
     }
 
     linkHighlight = function(url) {
-        var links = document.querySelectorAll('.navigation');
-        var activeLink = document.querySelector('.highlighted');
+        var links = document.getElementsByClassName('navigation');
```

Comment:
Please read the article :)

You shouldn't directly access the DOM even if it's technically possible.